screen-orientation/orientation-reading.html

```diff
@@ -53,7 +54,8 @@
   const secondaryOrientation2Angle = primaryOrientation2Angle === 90 ? 270 : 90;
   await screen.orientation.lock(orientations.secondaryOrientation2);
   assert_equals(
-    screen.orientation.angle == secondaryOrientation2Angle,
+    screen.orientation.angle,
+    secondaryOrientation2Angle,
     "Secondary orientation 2 angle must be the one of 90 or 270 not set to primary orientation 2"
   );
```

Comment:
The description here doesn’t seem to match the test very well... something doesn’t read well about it. Could you please rephrase it?

Reply:
I've given it a go, see what you think!

Reply:
Ah! much better!